pulumi/infra/config.py

```diff
@@ -9,14 +9,18 @@
 import pulumi
 
 # This will be incorporated into various infrastructure object names.
-DEPLOYMENT_NAME = pulumi.get_stack()
+DEPLOYMENT_NAME: Final[str] = pulumi.get_stack()
```

Comment:
Wonder if it's time to rename this to `STACK_NAME` instead of `DEPLOYMENT_NAME` to finally sever all ties with our CDK past?

(Doesn't need to be in this PR; just thinking aloud.)

Reply:
![image](https://user-images.githubusercontent.com/69007229/155190619-50bac4d0-df57-4760-afb5-9f4d364286bf.png)